### ISSUE TITLE
Bug with cell collections

### ISSUE THREAD

**Issue by dwightguth** [kframework/k-legacy#1724]
```
require "domains.k"

module TEST

  configuration <T> <threads multiplicity="?"> <thread multiplicity="*"> <k> $PGM:K </k> </thread>  </threads> </T>

  rule <threads> .Bag </threads> => .Bag

endmodule
```

```
kompile --kore test.k
krun -cPGM=5 --kore
```

**Comment by grosu** [comment]
Not sure I understand from this description what the issue really is, but it looks like something that should work because it is commonly used in our tutorial examples.  So I'm going to add it to our list of milestones and I hope that it takes a very simple fix.  @andreistefanescu , is this complicated?  Do you need anybody to help with this?

**Comment by andreistefanescu** [comment]
fixed in #1737
